src/sentry/static/sentry/app/components/substances/assignToWorkflow.jsx

```diff
@@ -3,30 +3,36 @@ import React from 'react';
 import createReactClass from 'create-react-class';
 import Modal from 'react-bootstrap/lib/Modal';
 import {connect} from 'react-redux';
+import ClimsTypes from 'app/climsTypes';
 
 import {t} from 'app/locale';
 import SelectControl from 'app/components/forms/selectControl';
-import {processDefinitionsGet} from 'app/redux/actions/processDefinition';
-import {processesPost} from 'app/redux/actions/process';
 
 import JsonForm from 'app/views/settings/components/forms/jsonForm';
 import LoadingIndicator from 'app/components/loadingIndicator';
 import Link from 'app/components/link';
 import Bpmn from 'app/components/bpmn';
 import styled from 'react-emotion';
+import {sortBy} from 'lodash';
+import {getPresetsById} from 'app/redux/selectors/processDefinition';
 
 import {
   processAssignSelectPreset,
   processAssignSelectProcess,
   processAssignSetVariable,
-} from 'app/redux/actions/processAssign';
+  processAssignmentsPost,
+  processAssignmentsSetEditing,
+} from 'app/redux/actions/processAssignment';
+
+import {processDefinitionActions} from 'app/redux/actions/processDefinition';
```

Comment:
Throughout this file, "process" is used. But the name of the file is "assignToWorkflow".

Reply:
Yes :( I'm sorry. I hope that it will be easier to decide on all of these names after I've gotten the diagrams down!